### PR TITLE
updated README with HTTP details

### PR DESCRIPTION
**Pull request by t-keazirian**
* added variable for port

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,21 +17,29 @@ dotnet --version
...
 
-- The server should establish a socket connection with the client using a low-level socket library. The goal of this exercise is to work with sockets directly.
-- The server should accept and return streams of data rather than raw strings.
-- The echo server should be covered by a robust suite of tests.
+## Implementation Requirements
+
+* The server should establish a socket connection with the client using a low-level socket library.
+* The server should accept and return streams of data rather than raw strings.
+* The HTTP server should be covered by a robust suite of unit tests.
+* The HTTP server should pass all of the tests covered in `01_getting_started` in the [HTTP Server Spec](https://github.com/8thlight/http_server_spec).
```
Comment (Aoxian) [inline README.md:40]: Pretty sure you can just link to the code in your repo.

```suggestion
* The HTTP server should pass all of the tests covered in `01_getting_started` in the [HTTP Server Spec](http_server_spec/features/01_getting_started).
```

File: README.md [hunk 2]
```diff
@@ -55,22 +63,23 @@ dotnet test
...
 
-#### Steps to Run
-1. Start your HTTP server on port `5000`
-2. `cd` into the `http_server_spec` folder:
-```shell
-cd http_server_spec 
-```
-3. Run the acceptance tests:
+### Run the Acceptance Tests
+Note: This server is currently programmed to run on port `5000`. If you would like to change the port, you can change the value of the variable, `private static int _port = 5000` (line 11) in `Server.cs` and recompile. In the future, the goal is to make this customizable via a command-line argument.
```
Comment (Aoxian) [inline README.md:70]: here you should definitely reference the code directly in a link.

```suggestion
Note: This server is currently programmed to run on port `5000`. If you would like to change the port, you can change the value of the variable `_port` in [Server.cs](TKeazirian.HTTPServer/Server.cs) and recompile. In the future, the goal is to make this customizable via a command-line argument.
```

File: README.md [hunk 3]
```diff
@@ -55,22 +63,23 @@ dotnet test
...
+
+1. Start your HTTP server (you can do this by running the `dotnet run --project TKeazirian.HTTPServer` command or click `Run` if you are using an IDE)
+2. Navigate into the `http_server_spec` folder: `cd http_server_spec`
```
Comment (Aoxian) [inline README.md:73]:
```suggestion
2. Navigate into the [http_server_spec](http_server_spec/) folder: 
```sh
cd http_server_spec
```
```

File: README.md (hunk 1 above)
Comment (t-keazirian) [reply README.md:40]: This is really cool - I was trying to figure out how to do this/if it was even possible so yay, learned something new and fun!

**Review by Aoxian — APPROVED** [review]

**Pull request merged**